core/templates/dev/head/pages/library/search_bar_directive.html

```diff
@@ -215,6 +215,7 @@
   /* This rule targets only Firefox browsers. */
   @-moz-document url-prefix() {
     .oppia-search-bar-category-selection-symbol {
+      padding-top: 16px;
```

Comment:
@vibhor98, This seems to only get reflected to Firefox browsers, have you tested this with other browsers? Also, I'm not sure about the issue #6111, can you please make confirm that this issue is at least reproducible, if so then in which browser?

Reply:
Can be merged as this issue was only related to Firefox (see [here](https://github.com/oppia/oppia/issues/6111#issuecomment-460265350)).

Reply:
Yeah, I found it just for Firefox.